yarn.lock

```diff
@@ -181,17 +181,49 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@a4u/a4u-collection-react-spectrum-open-source-color-icons-release@npm:latest":
```

Comment:
Do not commit the a4u assets in the lock file. It will cause issues in CI for one and prevent non-employees contributing to SWC.